### ISSUE TITLE
[BUG] QML Tiles not showing up on after connection is established on latest master commit

### ISSUE THREAD

**Issue by lifof** [cagnulein/qdomyos-zwift#362]
Is it just me or is it the case for everyone ? the app is loading just fine and all services are working, but something is holding the tiles from showing up on my computer.

![image](https://user-images.githubusercontent.com/1725688/121824697-fc548600-cca5-11eb-960d-9cd9bc79dc97.png)

 - OS: Manjaro Linux
 - Version 5.12.8

**Comment by cagnulein** [comment]
It seems that your pc can't find any Bluetooth devices. Attach the debug log, please

**Comment by lifof** [comment]
Here you go
[debug-Mon_Jun_14_00_17_27_2021.log](https://github.com/cagnulein/qdomyos-zwift/files/6647132/debug-Mon_Jun_14_00_17_27_2021.log)

**Comment by lifof** [comment]
> It seems that your pc can't find any Bluetooth devices. Attach the debug log, please

I don't think that's the case cause it made a connection with Zwift and I was able to do a 30 mins session yesterday, but as I said the tiles on the app interface were not showing up.

**Comment by cagnulein** [comment]
did you try the master instead of your branch? i run on a raspberry yesterday and it was fine (the master i mean)

**Comment by lifof** [comment]
Yes I did and it had the same thing on screen, I thought maybe It's a config that is doing that and tried to reset everything but I couldn't figure out the path where the config file is stored on my machine, finally it was late and I had to do my 30 mins session, so I gave up when I found that it's working even with the tiles not showing.

**Comment by cagnulein** [comment]
oh yeah now i remember! on some fast pc the bluetooth connection event comes first of the loading of the homeform.cpp
In order to confirm this, try to put a deviceConnected() at the end of the homeform::homeform(QQmlApplicationEngine* engine, bluetooth* bl)

of course this is not a solution, it's just a workaround. I think the right solution could be starting the bluetooth search after this connection 

connect(bluetoothManager, SIGNAL(deviceConnected()), this, SLOT(deviceConnected()));
